### PR TITLE
Fix lint issues on readme for authz tests

### PR DESCRIPTION
**Pull request by marcushines**

### CODE REVIEW

File: feature/security/gnsi/authz/tests/README.md [hunk 1]
```diff
@@ -1,17 +1,19 @@
...
 ## Summary
+
 Test gNSI API behaviors and gRPC authorization policy behaviors.
 
 ## Baseline Setup
 
 ### Input Args
 
-* `test_infra_id`: the SPIFFI ID that used by test infra clients.
+* `test_infra_id`: the SPIFFE-ID that used by test infra clients.
```
Comment (greg-dennis) [inline feature/security/gnsi/authz/tests/README.md:11]: Should SPIFFE-ID be hyphenated?

Comment (greg-dennis) [inline feature/security/gnsi/authz/tests/README.md:11]: "that used" -> "that is used"

**Review by greg-dennis — DISMISSED** [review]

File: feature/security/gnsi/authz/tests/README.md [hunk 2]
```diff
@@ -276,44 +279,45 @@ cert_read_only |deny |allow |deny |allow |deny |deny |deny |allow |deny
...
-  * Everytime a valid policy is automatically rolled back, the following OC leaves should be validated:
-    * `/system/grpc-servers/grpc-server/state/authz-policy-version` = `UploadRequest.version` of the previous request (the one rollback to).
-    * `/system/grpc-servers/grpc-server/state/authz-policy-created-on` = `UploadRequest.created_on` of the previous request (the one rollback to).
-  * An invalid policy should not trigger the following OC leaf updates:
-    * `/system/grpc-servers/grpc-server/state/authz-policy-version`
-    * `/system/grpc-servers/grpc-server/state/authz-policy-created-on`
-
-### Authz-1, test policy behaviors, and probe results matches actual client results.
+
+* Everytime a gRPC call (including gNSI calls themselves) is allowed or denied, the following OC leaves should be validated:
```
Comment (robshakir) [inline feature/security/gnsi/authz/tests/README.md:286]:
```suggestion
* Every time a gRPC call (including gNSI calls themselves) is allowed or denied, the following OC leaves should be validated:
```

File: feature/security/gnsi/authz/tests/README.md [hunk 3]
```diff
@@ -276,44 +279,45 @@ cert_read_only |deny |allow |deny |allow |deny |deny |deny |allow |deny
...
+  * `/system/grpc-servers/grpc-server/authz-policy-counters/rpcs/rpc[name]/state/name` is the matched request path, e.g. "/gribi.gRIBI/Get"
+  * `/system/grpc-servers/grpc-server/authz-policy-counters/rpcs/rpc/rpc[name]/state/access-accepts` increments if the rpc call is allowed.
+  * `/system/grpc-servers/grpc-server/authz-policy-counters/rpcs/rpc/rpc[name]/state/access-rejects` increments if the rpc call is denied.
+  * `/system/grpc-servers/grpc-server/authz-policy-counters/rpcs/rpc/rpc[name]/state/last-access-accept` reflects the timestamp of the method call.
+  * `/system/grpc-servers/grpc-server/authz-policy-counters/rpcs/rpc/rpc[name]/state/last-access-reject` reflects the timestamp of the method call.
+* Everytime a valid policy is pushed (even it's not finalized), the following OC leaves should be validated:
```
Comment (robshakir) [inline feature/security/gnsi/authz/tests/README.md:292]:
```suggestion
* Every time a valid policy is pushed (even it's not finalized), the following OC leaves should be validated:
```

File: feature/security/gnsi/authz/tests/README.md [hunk 4]
```diff
@@ -276,44 +279,45 @@ cert_read_only |deny |allow |deny |allow |deny |deny |deny |allow |deny
...
+  * `/system/grpc-servers/grpc-server/state/authz-policy-version` = `UploadRequest.version` in the API proto.
+  * `/system/grpc-servers/grpc-server/state/authz-policy-created-on` = `UploadRequest.created_on` (in terms of represented time).
+* Everytime a valid policy is automatically rolled back, the following OC leaves should be validated:
```
Comment (robshakir) [inline feature/security/gnsi/authz/tests/README.md:295]:
```suggestion
* Every time a valid policy is automatically rolled back, the following OC leaves should be validated:
```

**Review by robshakir — DISMISSED** [review]

**Review by greg-dennis — APPROVED** [review]

**Pull request merged**